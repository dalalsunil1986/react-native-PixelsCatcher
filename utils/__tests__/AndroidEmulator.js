jest.mock('child_process', () => ({ spawn: jest.fn() }));
jest.mock('../exec', () => jest.fn());
jest.mock('../delay', () => jest.fn());
jest.mock('../log', () => ({ v: jest.fn(), d: jest.fn() }));

const AndroidEmulator = require('../AndroidEmulator');
const exec = require('../exec');

describe('AndroidEmulator', () => {
  const name = 'emulator_name';

  it('initialise emulator', async () => {
    exec.mockImplementationOnce(() => 'avd devices');

    const emularor = new AndroidEmulator(name);
    expect(emularor).toMatchSnapshot();
  });
});
