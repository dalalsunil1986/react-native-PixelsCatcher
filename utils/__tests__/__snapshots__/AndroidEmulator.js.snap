// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AndroidEmulator initialise emulator 1`] = `
AndroidEmulator {
  "_name": "emulator_name",
}
`;
